frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -314,98 +335,114 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
   };
 
   const [formValues, setFormValues] = React.useState<FormValues>({
-    'metadata.name': 'example',
-    'metadata.labels': [],
+    'metadata.name': _.get(sample, 'metadata.name', 'example'),
```

Comment:
This line is what was causing the integration test failure. Previously, `metadata.name` was not synced from the YAML view and just defaulted to 'example', so when the integration tests created a new operand using the form, it didn't conflict with the existing one. Now that data is synced, the 'metadata.name' field from the YAML view is carried over to the form view and conflicts with the existing resource.

Reply:
@benjaminapetersen @spadgett I guess an easier solution to this failure would be to add a step to change the operand name before we create it via the form. But... since deleting and recreating it seems to work just fine (and adds a little bit more coverage), we might as well just leave it. Any objections or thoughts?